tests/subsys/audio_modules/audio_module_template/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  nrf5340_audio.audio_module_template:
+    sysbuild: true
+    platform_allow: qemu_cortex_m3
+    integration_platforms:
+      - qemu_cortex_m3
+    tags: audio_module audio_module_template nrf5340_audio_unit_tests sysbuild
```

Comment:
@nordic-piks please take a look at tags

Reply:
Please add CI tag which will allow to skip those tests.
e.g. ci_tests_subsys_audio_modules at https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/ci/tags.yaml
and define its dependencies.